src/main.rs

```diff
@@ -1,6 +1,31 @@
 // SPDX-FileCopyrightText: 2022 Profian Inc. <opensource@profian.com>
 // SPDX-License-Identifier: AGPL-3.0-only
 
+//!
+//!# Overview
+//!
+//!Steward is a critical element of the Confidential Computing infrastructure. The promise of Confidential Computing is fully utilized when the workload runtime (Enarx WebAssembly) deployed into a Trusted Execution Environment (TEE) is assessed and verified for correctness before an actual workload is released into a TEE from the registry (Drawbridge). An external **attestation service** must perform evidence verification and assessment of the hardware's trustworthiness.
```

Comment:
1. You need a space after `//!`.
2. Lines should be wrapped at 80 characters.

Reply:
Do this for all the new documentation.

Reply:
The change is made.